winterfell/controllers/fileClaimController.js

```diff
@@ -29,7 +30,7 @@ module.exports = function (app) {
   });
 
   // Get a particular claim
-  app.get('/claims/:extClaimId', function (req, res) {
+  app.get('/claim/:extClaimId', auth.authenticatedOr404, function (req, res) {
```

Comment:
well what do you think? We can keep it singular `/claim/:claimId`, but we should be consistent across the board.